### ISSUE TITLE
Properties located in water bodies

### ISSUE THREAD

**Issue by kristydahl** [zillow-research/ztrax#2]
I'm working with the ZTRAX data in coastal states and have noticed that, no matter which coordinate reference system I use to project the data, there's a systematic misalignment of *some* coastal properties such that some are located in coastal waterbodies. Others simply don't align with the streets they are supposedly on when I compare their locations to other geographic datasets. 

When I compare the lat/long coordinates of these properties from within ZTRAX, they don't match the lat/long coordinates you get when you locate them via, for example, Google Maps. 

Is there something I'm not aware of about the way these were geocoded? Are they in WGS1984? Or something I'm not understanding about how to project the data? 

Attaching an example of the issue from Connecticut. Any help much appreciated!

<img width="1384" alt="ct_ztrax_properties_issue" src="https://user-images.githubusercontent.com/2522257/31787050-5a824b34-b4bf-11e7-8eb0-79a338cf7989.png">

**Comment by chrissip-zillow** [comment]
See this FAQ. It addresses this issue.

Skylar

**Comment by kristydahl** [comment]
Thanks, Skylar. I'm not seeing a link to the FAQ you mentioned. Can you resend the link?

**Comment by chrissip-zillow** [comment]
Attached. Are the attachments not making it through?  Can you send your local email?

**Comment by kristydahl** [comment]
Yeah, I'm not seeing attachments. My email is kristina.a.dahl@gmail.com. 

Thank you!

**Comment by kristydahl** [comment]
For future reference, this is Zillow's response to this question in their ZTRAX FAQ: 

"How accurate are the Latitude and longitudes coordinates in ZTRAX?
As far as the geocodes, these are enhanced Tiger coordinates. They are different from the roof top geocodes that are parcel centroids that we use on the site. We do not have redistribution rights to the parcel centroids used on the site. The coordinates provided are interpolated. They are placed on a block segment by address. For example, in 100-199 block range, 150 would be in the middle. These are more useful for measuring distance between properties, not as useful in rendering on a map. Most users re-geocode the properties using the property addresses and a third party mapping service for display."
